doc-surrealdb_versioned_docs/version-1.x/surrealql/parameters.mdx

```diff
@@ -30,103 +62,221 @@ let people = await surreal.query("SELECT * FROM article WHERE status INSIDE $sta
-                <code>$scope</code>
-            </td>
-            <td colspan="2" scope="row" data-label="Description">
-                Represents the name of the scope of a currently authenticated scope user.
-            </td>
-        </tr>
-        <tr>
-            <td colspan="2" scope="row" data-label="Name">
-                <code>$session</code>
-            </td>
-            <td colspan="2" scope="row" data-label="Description">
-                Represents values from the session functions as an object
-            </td>
-        </tr>
-        <tr>
-            <td colspan="2" scope="row" data-label="Name">
-                <code>$before</code>
-            </td>
-            <td colspan="2" scope="row" data-label="Description">
-                Represents the value before a mutation on a field
-            </td>
-        </tr>
-        <tr>
-            <td colspan="2" scope="row" data-label="Name">
-                <code>$after</code>
-            </td>
-            <td colspan="2" scope="row" data-label="Description">
-                Represents the value after a mutation on a field
-            </td>
-        </tr>
-        <tr>
-            <td colspan="2" scope="row" data-label="Name">
-                <code>$value</code>
-            </td>
-            <td colspan="2" scope="row" data-label="Description">
-                Represents the value after a mutation on a field (identical to $after in the case of an event)
-            </td>
-        </tr>
-        <tr>
-            <td colspan="2" scope="row" data-label="Name">
-                <code>$input</code>
-            </td>
-            <td colspan="2" scope="row" data-label="Description">
-                Represents the initially inputted value in a field definition, as the value clause could have modified the $value variable
-            </td>
-        </tr>
-        <tr>
-            <td colspan="2" scope="row" data-label="Name">
-                <code>$this</code>
-            </td>
-            <td colspan="2" scope="row" data-label="Description">
-                Represents the current record in a subquery
-            </td>
-        </tr>
-        <tr>
-            <td colspan="2" scope="row" data-label="Name">
-                <code>$parent</code>
-            </td>
-            <td colspan="2" scope="row" data-label="Description">
-                Represents the parent record in a subquery
-            </td>
-        </tr>
-         <tr>
-            <td colspan="2" scope="row" data-label="Name">
-                <code>$event</code>
-            </td>
-            <td colspan="2" scope="row" data-label="Description">
-                Represents the type of table event triggered on an event.
-            </td>
-        </tr>
-    </tbody>
-</table>
\ No newline at end of file
+SurrealDB uses predefined variables. For that purpose, you can use those variables inside your query but you cannot declare new parameters using one of the following names:
```

Comment:
question: should we regroup reserved variables blocks by themes? I do not see a particular theme but maybe you do.

Reply:
In the PR as it stands they are almost all taken from the previous table with a bit of combining ($before and $after, etc.) so haven't thought about theme, but I do see one when I reread it:

1) Auth, scope, etc. which are about database config
2) $before, $after and the rest which are about queries and mutation
3) Finally $event which feels a bit separate.

I could see it making sense to move the query/mutation ones up top though (so 2, then 1, then 3) since those examples are both more instructive, frequent, and less intimidating. So I think I'll do that.